language/en-GB/lib_joomla.ini

```
@@ -448,6 +448,7 @@ JLIB_HTML_NO_RECORDS_FOUND="No records found."
 JLIB_HTML_PAGE_CURRENT="Page %s"
 JLIB_HTML_PAGE_CURRENT_OF_TOTAL="Page %s of %s"
 JLIB_HTML_PAGINATION="Pagination"
+JLIB_HTML_PAGINATION_NUMBERS="%d - %d / %d items"
```

```suggestion
JLIB_HTML_PAGINATION_NUMBERS="%1$d- %2$d/ %3$d items"
```

Without this translators are unable to change the order of the variables which might be required in their language

---

I thought the same but didn't want to confuse translators. We don't use the syntax anywhere in the core, sadly... Not sure if we should introduce it here or better in the translation documentation? Do we have such documentation?

---

we do use it  (over 900 times) - just not as regularly as we should

and it is documented https://docs.joomla.org/Formatted_fields_in_language_translation_strings/en

---

oh, my quick search had the wrong query string, so yes we use it, then I merge this but I think your suggestion is wrong. Your suggestion would be `1- 20/ 50 item` without spaces, that's not intended right?

---

yes that was a typo - i have amended it now